### ISSUE TITLE
Frontend Design

### ISSUE THREAD

**Issue by karkir0003** [DSGT-DLP/Deep-Learning-Playground#4]
React App frontend where user can drag and drop model architecture (pieces that are draggable for now are `Softmax`, `Conv2d`, `Linear `(ie: hidden layer of NN), `ReLU`, `Sigmoid`, `BatchNorm`). For each of these draggable pieces, the users should be able to toggle relevant parameters (if any). For example, `Conv2d` parameters users can wiggle around with are in_channels, out_channels, kernel_size, stride. Given the pieces user drags and drop into the pallette/plate, drag and drop should render a string of the list of dragged pieces and that gets passed to the backend. Pay attention to the backend schema of how each layer should be read as a string (in particular, don't forget the `nn` before each layer). For example, `nn.Linear(in_features=50, out_features=30)` GOOD, but `Linear(in_features=50, out_features=30)` BAD
